### PR TITLE
create server - mvc structure - gitignore

### PR DESCRIPTION
**Pull request by ghost**
relates #21

### CODE REVIEW

**Review by veredrec — APPROVED** [review]

**Review by KatiaAsh — APPROVED** [review]
:fire:

**Review by Heathercoraje — APPROVED** [review]
looking good

File: package.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    "express-handlebars": "^3.0.0",
+    "serve-favicon": "^2.4.3"
+  },
+  "devDependencies": {
+    "istanbul": "^0.4.5",
+    "nodemon": "^1.11.0",
+    "shot": "^3.4.2",
+    "tap-spec": "^4.1.1",
+    "tape": "^4.8.0"
+  }
```
Comment (philawsophizing) [inline package.json:21]: Are you using all these dependencies? Best to require them as they are needed.

File: package.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
+{
+  "name": "Girls-n-a-Dude",
+  "version": "1.0.0",
+  "author": "",
+  "license": "MIT",
+  "scripts": {
+     "start": "node src/server.js",
+    "start:watch": "nodemon -q src/server.js"
+  },
```
Comment (philawsophizing) [inline package.json:9]: indentation is different for "start" and "start:watch". I also prefer "dev" over start:watch (more clear) but that's preference :smile:

**Review by philawsophizing — CHANGES_REQUESTED** [review]
Two very quick things, I'll merge this right away if you can correct them.

File: package.json (hunk 1 above)
Comment (veredrec) [reply package.json:21]: @philawsophizing thanks for the comments. We are changing the devStart, and we are planning to use all of these dependencies, if time allows us :wink:

**Pull request merged**